whitepapers/ingest/Graph.md

```diff
@@ -0,0 +1,29 @@
+# MutableGraph [NOT IMPLEMENTED]
```

Comment:
Oh, using our own objects as nodes/edges is pretty neat. We'd just have to decide how to translate our schema to/from a networkx graph (are relations/throughs edges or nodes?). I still worry a general-purpose graph library could make implementing regulator steps unnecessarily complicated, but if that happens I'm sure we could smooth things over with some convenience methods or something.

I don't remember why we decided against it in the first place, though.